src/components/notifications/SIGNL4.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+    <div class="mb-3">
+        <label for="webhook-url" class="form-label">{{ $t("SIGNL4 Webhook URL") }}</label>
+        <input
+            id="webhook-url"
+            v-model="$parent.notification.webhookURL"
+            type="url"
+            pattern="https?://.+"
+            class="form-control"
+            required
+        />
+    </div>
+
+    <div class="mb-3">
+        <label for="webhook-request-body" class="form-label">{{
+            $t("Alert Body")
+        }}</label>
+        <select
+            id="webhook-request-body"
+            v-model="$parent.notification.webhookContentType"
+            class="form-select"
+            required
+        >
+            <option value="default">{{ $t("signl4WebhookDefault", ["application/json"]) }}</option>
+            <option value="custom">{{ $t("signl4WebhookCustom") }}</option>
+        </select>
+
+        <div class="form-text">
+            <div v-if="$parent.notification.webhookContentType == 'default'">
+                <p>{{ $t("signl4WebhookDefaultDesc", ['"application/json"']) }}</p>
+            </div>
+            <div v-if="$parent.notification.webhookContentType == 'custom'">
+                <i18n-t tag="p" keypath="signl4WebhookCustomDesc">
```

Comment:
Please extract this into a shared component between the webhook and this component.
They should not need different docs.

Reply:
Not sure what you mean here? Do you mean the strings? I guess there are some differences between Webhook and SIGNL4.

Reply:
I mean that something like the following is not quite useful for users to get how templating works.


```
    "signl4WebhookCustomDesc": "Send a custom alert data"
````

Instead, If the code documenting how templating works could be moved into `TemplatedInput.vue`
=> Allows documenting this only once and quality is likely better too..

Reply:
Ah, got it. Thanks a lot.

After some contemplation I have decided to omit the templates. I think this causes more confusion than benefits.

I will update the code now.

Thanks a lot for your help. This is a really cool tool.